src/dfx/src/actors/mod.rs

```diff
@@ -63,7 +70,12 @@ pub fn start_emulator_actor(
     // handing it over to ic-ref. If we read the file and it has
     // contents we shall assume it is due to our spawned ic-ref
     // process.
-    std::fs::write(&emulator_port_path, "")?;
+    std::fs::write(&emulator_port_path, "").with_context(|| {
+        format!(
+            "Failed to write/clear emulator port file {:?}.",
+            &emulator_port_path
```

Comment:
Is it more common/accepted to use the debug format, or to use .to_string_lossy()?
```suggestion
            "Failed to write/clear emulator port file {}.",
            emulator_port_path.to_string_lossy()
```

Reply:
Looks like we use `to_string_lossy()` more often. I'll switch to that in the messages